README.md

```diff
@@ -302,9 +302,13 @@ dpkg -P chefdk
 
 To build the chef-dk, we use the omnibus system. Go to the [omnibus README](omnibus/README.md) to find out how to build!
 
-To update the chef-dk's dependencies, run `rake dependencies`. This will update the `Gemfile.lock`, `Gemfile.windows.lock`, `omnibus/Gemfile.lock`, `acceptance/Gemfile.lock`, `omnibus/Berksfile.lock`, and `omnibus_overrides.rb`.  It will also show you any outdated dependencies due to conflicting constraints. Some outdated dependencies are to be expected; it will inform you if any new ones appear that we don't know about, and tell you how to proceed.
+## Updating Dependencies
 
-To add or remove a package from the chef-dk, edit `Gemfile`.
+To update the chef-dk's dependencies, run `rake bundle:update` (or `rake bundle:install`). To update specific gems only, run `rake bundle:update[gem1 gem2 ...]`. `bundle update` and `bundle install` will *not* work, on purpose: the rake task handles both the windows and non-windows lockfiles and updates them in sync.
+
+To perform a full update of all dependencies everywhere, run `rake dependencies`. This will update the `Gemfile.lock`, `Gemfile.windows.lock`, `omnibus/Gemfile.lock`, `acceptance/Gemfile.lock`, `omnibus/Berksfile.lock`, and `omnibus_overrides.rb`.  It will also show you any outdated dependencies due to conflicting constraints. Some outdated dependencies are to be expected; it will inform you if any new ones appear that we don't know about, and tell you how to proceed.
+
+To add or remove a package from the chef-dk, edit `Gemfile`. To change binary versions, edit `version_policy.rb`.
 
```

Comment:
I don't see any `Gemfile.local`, but there is no reason that won't work.

You can bundle install and update from omnibus and acceptance as usual, if you like.